### ISSUE TITLE
can't reproduce  streaming_conformer_ctc result

### ISSUE THREAD

**Issue by BuaaAlban** [k2-fsa/icefall#909]
Hi @glynpu ,
Could you share your  config to reproduce your trained model? 
I can't reproduce the result using the default values in `streaming_conformer_ctc/train.py` , after training for 46 epochs and decode using it with avg=20, acutually there is almost no acc with the config
BTW, there are some small bugs like missing ids in the result tuple and dataloader error when using my trained model to decode `streaming_conformer_ctc/streaming_decode.py`

**Comment by glynpu** [comment]
Thanks for the report.
> there are some small bugs like missing ids in the result tuple and dataloader error

Q1: Is it possible to decode with these trained models with your dataset? 
https://huggingface.co/GuoLiyong/streaming_conformer/tree/main/streaming_models


Q2: Do you have some special reasons to use this ctc recipe? Since we already shift to RNN_T loss, so we are not working on this direction recently.
But if you really need a model trained with CTC Loss. We are glad to continue optimize this.

**Comment by glynpu** [comment]
> maybe I can create a PR later.

Thanks!

> and maybe easier for deployment

For RNN-T,  our [sherpa repo](https://github.com/k2-fsa/sherpa) is mainly aiming for RNN-T models deployment, maybe it will help you to deploy an RNN-T model.
For CTC models, currently and in the short future, we are not going to support it as much as RNN-T models.

**Comment by danpovey** [comment]
Some of the older conformer models could sometimes completely fail to converge.  For more recent models we have made various changes that make convergence much more stable. 
The loss should be (0.0x) or (0.00x), if it's, like, 0.5 or 0.8 it means it completely failed to converge.
